Anytype/Sources/PresentationLayer/Flows/EditorPageFlow/EditorPageCoordinatorViewModel.swift

```diff
@@ -1,27 +1,42 @@
 import Foundation
 import SwiftUI
+import AnytypeCore
+import Services
 
 @MainActor
-final class EditorPageCoordinatorViewModel: ObservableObject, EditorPageModuleOutput {
+final class EditorPageCoordinatorViewModel: ObservableObject, EditorPageModuleOutput, RelationValueCoordinatorOutput {
     
     private let data: EditorPageObject
     private let showHeader: Bool
     private let setupEditorInput: (EditorPageModuleInput, String) -> Void
     private let editorPageAssembly: EditorPageModuleAssemblyProtocol
+    private let legacyRelationValueCoordinator: LegacyRelationValueCoordinatorProtocol
+    private let relationValueCoordinatorAssembly: RelationValueCoordinatorAssemblyProtocol
+    private let relationValueProcessingService: RelationValueProcessingServiceProtocol
+    private let toastPresenter: ToastPresenterProtocol
```

Comment:
ToastPresenterProtocol will be delete. Try to using `view.snackbar(` in new places

Reply:
thanks, I'll try 👌